operator/src/main/java/oracle/kubernetes/operator/DomainStatusUpdater.java

```diff
@@ -1029,7 +1028,24 @@ private boolean isNonClusteredServer(String serverName) {
       }
 
       private boolean isRunning(@Nonnull String serverName) {
-        return RUNNING_STATE.equals(getRunningState(serverName));
+        return RUNNING_STATE.equals(getRunningState(serverName)) && isRollCompleteFor(serverName);
```

Comment:
Why is running tied to whether or not roll is complete? (I assume "isRunning" feeds into the running status reporting, which makes this change suspect. If the change isn't suspect, can this be changed to "isRunningAndRollComplete" or some-such.)

Reply:
If the roll is not complete, the pod is still running an obsolete configuration, and therefore does not count as completed. The name is likely unclear, though, and should be changed.